events/tests.py

```diff
@@ -0,0 +1,64 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from .models import Event
+from datetime import datetime
+
+import pytest
+
+
+DATE_TIME_START = datetime(2022, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+DATE_TIME_END = datetime(2022, 3, 24, 14, 12, 12, 0, tzinfo=timezone.utc)
+TITLE = 'new_title'
+LOCATION = 'new_location'
+DESCRIPTION = 'new_description'
+
+
+@pytest.fixture
+def new_event():
+    return Event(title=TITLE, location=LOCATION, description=DESCRIPTION,
+                 date_time_start=DATE_TIME_START, date_time_end=DATE_TIME_END)
```

Comment:
The convention is to open the parentheses and create a new line then add what you need and close the parenthesis in a new line - in this way it is more visible where it start and end
```suggestion
    return Event(
        title=TITLE, location=LOCATION, description=DESCRIPTION,
        date_time_start=DATE_TIME_START, date_time_end=DATE_TIME_END
    )
```

Reply:
done